anscii_symbols.c

```diff
@@ -0,0 +1,4 @@
+const int newLine = 10;
```

Comment:
Please make them Caps Lock, so it would be clear that they aren't just variables.